index.js

```diff
@@ -43,17 +38,32 @@ module.exports = (robot, options, visit) => {
     });
   }
 
+  // Trigger the event
+  function trigger(event) {
+    robot.webhook.emit(event.event, event);
+  }
+
   function schedule(installation, repository) {
     // Wait a random delay to more evenly distribute requests
     const delay = options.delay ? options.interval * Math.random() : 0;
 
     robot.log.debug({repository, delay, interval: options.interval}, `Scheduling interval`)
 
     setTimeout(() => {
-      // Schedule visit to this repository on an interval
-      intervals[repository.id] = setInterval(() => visit(installation, repository), options.interval);
-      // Make the first visit now
-      visit(installation, repository);
+      const event = {
```

Comment:
![](https://upload.wikimedia.org/wikipedia/commons/thumb/1/14/Rubber_Duck_%288374802487%29.jpg/220px-Rubber_Duck_%288374802487%29.jpg)

continuing to talk to myself, this is unnecessary because it will be handled when this fake "event" gets processed by the normal events.on flow of (p)robot.

So I think this whole comment is OBE.